src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,370 @@
+updated_on: 2016-08-12
+authors:
+  - beaufortfrancois
+tags:
+  - news
+  - USB
+  - WebUSB
+  - IoT
+  - Arduino
+  - origintrials
+---
+
+If I said plain and simple "USB", there is a good chance that you will
+immediately think of keyboards, mice, audio, video and storage devices. You're
+right but you'll find other kinds of Universal Serial Bus (USB) devices out
+there.
+
+These non-standardized USB devices require hardware vendors to write native
+drivers and SDKs in order for you (the developer) to take advantage of them.
+Sadly this native code has historically prevented these devices from being used
+by the Web. And that's one of the reasons the WebUSB API has been created: to
+provide a way to expose USB device services to the Web. With this API, hardware
+manufacturers will be able to build cross-platform JavaScript SDKs for their
+devices.
+But most importantly this will **make USB safer and easier to use by bringing
+it to the Web**.
+
+Let's see what you could expect with the WebUSB API:
+
+1. Buy a USB device,
+2. Plug it into your computer,
+3. A notification appears right away, with the right website to go to for this device.
+4. Simply click on it. Website is there and ready to use!
+5. Click to connect and a USB device chooser shows up in Chrome, where you can pick your device.
+6. Tada!
+
+What could it have been after you've plugged your USB device?
+
+- Read a box, label, or search online for website -- possibly ending up on the wrong website.
+- Have to install a native application.
+- Is it supported on my operating system? Make sure you download the "right" thing.
+- Scary OS prompts popup and warn you about installing drivers/applications from the Internet.
+- Malfunctioning code harms the whole computer, the Web is built to [contain malfunctioning websites](https://www.youtube.com/watch?v=29e0CtgXZSI).
+- Only use the USB device once? On the Web, the website is gone once you closed tab. On a computer the code sticks around.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how USB works. If not, I
+recommend reading [USB in a NutShell](http://www.beyondlogic.org/usbnutshell).
+For background information about USB, check out the [official USB
+specifications](http://www.usb.org/home).
+
+The Chrome team is excited to be launching experimental features early to
+developers. The [WebUSB API](https://wicg.github.io/webusb/) is currently a
+draft which means that it is far enough along to be real and usable, but there
+is still time to make fixes that developers need. That's why the Chrome Team is
+actively looking for eager developers to give it a try and send
+[feedback on the spec](https://github.com/wicg/webusb/issues) and
+[feedback on the implementation](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EUSB).
+
+The WebUSB API is currently planned to be enabled experimentally on your
+origin in Origin Trials, and available now locally on your machine using an
```

Comment:
'planned to be enabled experimentally on your origin' 

I don't like the phrasing of this. It makes it sound like we've done something for them we haven't. Simply removing 'on your origin' would fix this.